deployment/bare_metal/build_frontend.sh

```diff
@@ -2,7 +2,9 @@
 # For that to work, node v12.19.0 needs to be installed with nvm for root
 # or NPM_BIN Path and NVM_DIR must be adjusted
 
-cd /var/www/html/gradido/frontend
+cd /var/www/html/gradido
+eval "echo \"$(cat .env.shell)\"" > .env
```

Comment:
May this override a `.env` coming from another source?

Reply:
yes it does, so I have added .env.local.dist 
For own .env entries you save .env.local.dist as .env.local and my deployment script copy this first then add the result of .env.shell